entities/project.ts

```diff
@@ -254,9 +234,9 @@ class Project extends BaseEntity {
 
     // Sorts
     if (sortBy === OrderField.Reactions) {
-      this.addReactionsCountQuery(query, direction);
+      query.orderBy(`project."totalHearts"`, direction);
     } else if (sortBy === OrderField.Donations) {
-      this.addTotalDonationsQuery(query, direction);
+      query.orderBy(`project."totalDonations"`, direction);
```

Comment:
I think since this are columns in the same entity instead of a relation. We can use the "else" query that orders by the attribute instead of another if.

Reply:
Changing the Enum to
```
export enum OrderField {
  CreationDate = 'creationDate',
  Balance = 'balance',
  QualityScore = 'qualityScore',
  Verified = 'verified',
  Reactions = 'totalReactions',
  Donations = 'totalDonations',
}
```

Reply:
Very good point, done.